apps/web/src/features/create-project/views/ProjectCreationWizard.tsx

```diff
@@ -0,0 +1,31 @@
+import { ProjectCreationStep } from "../application/createProject.reducer";
+import ProjectCreationConfirmation from "./confirmation";
+import ProjectTypesForm from "./project-types";
+import RenewableEnergyTypesForm from "./renewable-energy-types";
+import Stepper from "./Stepper";
+
+import { useAppSelector } from "@/shared/views/hooks/store.hooks";
+
+function ProjectCreationWizard() {
+  const projectCreationState = useAppSelector((state) => state.projectCreation);
```

Comment:
Les selector utilisent de la memoization under the hood mais si n'importe quel champ de `state.projectCreation` change, tu auras un re-render de ton composant alors qu'ici tu n'utilises que le champ `.step`.

Je ne suis pas un fou de sur-optimisation des re-renders React mais dans ce genre de cas ça ne coûte rien et ça rend plus explicite les informations du store que tu utilises.

`const projectCreationStep = useAppSelector((state) => state.projectCreation.step)`